src/index.jsx

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
```

Comment:
It is better to use another function: https://github.com/zalmoxisus/redux-devtools-extension#14-using-in-production